go.mod

```diff
@@ -20,115 +20,22 @@ require (
 require go.opentelemetry.io/collector/config/configopaque v1.3.0
 
 require (
```

Comment:
Was this a simple tidy?

Reply:
Yep, though I was surprised by this as well.